### DISCUSSION TITLE
primary key starting value

### DISCUSSION THREAD

**Discussion by avico78** [collerek/ormar#201] [Q&A]
Is there an option to have a starting value of primary key ?

```python
class Employee(ormar.Model):
    class Meta(BaseMetaSyntSrc):
        tablename: str = "employee"
    id: int = ormar.Integer(primary_key=True , starting_val=10000)
````

**Comment by collerek** [comment]
You cannot do this in `ormar` alone.

You can issue a separate query (for each table) with a start value like in: https://stackoverflow.com/questions/10494033/setting-sqlalchemy-autoincrement-start-value

Note that for some backends that natively supports Sequence it's apparently possible (as described in the same post) but this functionality has not been ported to `ormar` as it's backend specific and does not work with mysql and sqlite.
